beacon-chain/cache/proposer_indices_disabled.go

```diff
@@ -4,11 +4,26 @@
 package cache
 
 import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	forkchoicetypes "github.com/prysmaticlabs/prysm/v4/beacon-chain/forkchoice/types"
 	fieldparams "github.com/prysmaticlabs/prysm/v4/config/fieldparams"
 	"github.com/prysmaticlabs/prysm/v4/consensus-types/primitives"
 )
 
+var (
```

Comment:
Grammar nazi here, but both comments and help texts sound like it's the requests that are kept in the cache.

Reply:
Those were already present, I just copied them, will take any description you suggest.

Reply:
```
// ProposerIndicesCacheMiss tracks the number of requests when proposer indices aren't present in the cache.
ProposerIndicesCacheMiss = promauto.NewCounter(prometheus.CounterOpts{
	Name: "proposer_indices_cache_miss",
	Help: "The number of requests when proposer indices aren't present in the cache.",
})
// ProposerIndicesCacheHit tracks the number of requests when proposer indices are in the cache.
ProposerIndicesCacheHit = promauto.NewCounter(prometheus.CounterOpts{
	Name: "proposer_indices_cache_hit",
	Help: "The number of requests when proposer indices are present in the cache.",
})
```